src/gql/directives/WordsmithTransform.php

```diff
@@ -0,0 +1,341 @@
+<?php
+/**
+ * Wordsmith
+ *
+ * @author     Michael Rog <michael@michaelrog.com>
+ * @link       https://topshelfcraft.com
+ * @copyright  Copyright 2020, Top Shelf Craft (Michael Rog)
+ * @see        https://github.com/topshelfcraft/Wordsmith
+ */
+
+namespace topshelfcraft\wordsmith\gql\directives;
+
+use topshelfcraft\wordsmith\Wordsmith;
+
+use craft\gql\base\Directive;
+use craft\gql\GqlEntityRegistry;
+use GraphQL\Error\Error;
+use GraphQL\Language\DirectiveLocation;
+use GraphQL\Type\Definition\Directive as GqlDirective;
+use GraphQL\Type\Definition\ResolveInfo;
+use GraphQL\Type\Definition\Type;
+
+/**
+ * @author Michael Rog <michael@michaelrog.com>
+ * @package Wordsmith
+ * @since 3.3.0.1
+ */
+class WordsmithTransform extends Directive
+{
+
+
+    /**
+     * @inheritdoc
+     */
+    public static function create(): GqlDirective
+    {
+        if ($type = GqlEntityRegistry::getEntity(self::name())) {
+            return $type;
+        }
+
+        $type = GqlEntityRegistry::createEntity(static::name(), new self([
+            'name' => static::name(),
+            'locations' => [
+                DirectiveLocation::FIELD,
+            ],
+            'description' => 'Run Wordsmith transforms on the value of the field.',
+            'args' => [
+                'function' => [
+                    'name' => 'function',
+                    'type' => Type::string(),
+                    'description' => 'The Wordsmith function to use.'
+                ],
+                // 'args' => [
```

Comment:
Can we go ahead and accept an array of arguments via an `args` arg? 😆 (Say that five times fast.)

Sure, it won't be quite as nice as if those args were fully self-documenting via GQL... but... we can try to improve that later. For now, having all the functions available is probably better for consistency.